### PR TITLE
Add link to user guide

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
ping @gosonzhang @TisonKun

File: README.cn.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
+# TubeMQ
+
+[英文](README.md)
```
Comment (tisonkun) [inline README.cn.md]:
```suggestion
[英文版 README](README.md)
```

File: README.cn.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+
 TubeMQ是腾讯大数据在2013年开始研发的分布式消息中间件系统（MQ），专注服务大数据场景下海量数据的高性能存储和传输。经过近7年上万亿的海量数据沉淀，较之于众多的开源MQ组件，TubeMQ在海量实践（稳定性+性能）和低成本方面有一定的优势，近期我们在开源TubeMQ的相关代码及设计，更多资料正在陆续整理和上传之中。
+
+[用户手册](./docs/tubemq_user_guide.md)
```
Comment (tisonkun) [inline README.cn.md]:
```suggestion
## 文档
- [用户手册](./docs/tubemq_user_guide.md)
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
+# TubeMQ
+
+[中文](README.cn.md)
```
Comment (tisonkun) [inline README.md]:
```suggestion
[README in Chinese](README.cn.md)
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
+
 TubeMQ is a distributed messaging queue (MQ) system developed by Tencent Big Data since 2013. It focuses on high-performance storage and transmission of massive data in big data scenarios.
 
 After nearly seven years of massive data precipitation, TubeMQ has certain advantages in production practice (stability + performance) and low cost compared to many open source MQ projects. Recently we open source related code and design for TubeMQ project. You can find more information and documentations under /docs. More information and documentation (in English) will get uploaded soon.
 
+[User Guide](./docs/tubemq_user_guide.md)
```
Comment (tisonkun) [inline README.md]:
```suggestion
## Documentation
- [User Guide](./docs/tubemq_user_guide.md)
```

**Review by tisonkun — CHANGES_REQUESTED** [review]
Thanks for your contribution @netroby! I think it is really helpful in documentation side. I leaves some suggestions for pretty format that you might adopt. Let me know if you have other thoughts ;-)

**Comment by ghost** [comment]
Suggestion apply

**Review by tisonkun — APPROVED** [review]
Thanks for your update. LGTM.

Merging...

**Pull request merged**